csharp/ql/lib/semmle/code/csharp/dataflow/internal/DataFlowPrivate.qll

```diff
@@ -1485,6 +1491,13 @@ class FieldOrProperty extends Assignable, Modifiable {
     or
     result.(PropertyContent).getProperty() = this.getUnboundDeclaration()
   }
+
+  /** Gets the initializer of this field or property, if any. */
+  Expr getInitialier() {
```

Comment:
Typo: `getInitialier` should be `getInitializer`.

Reply:
Turns out we actually don't need it.